setup.py

```diff
@@ -96,12 +96,13 @@ def _get_cmdclass():
 def spirv_compile():
     if IS_LIN:
         compiler = os.path.join(
-            os.environ.get("ONEAPI_ROOT"), "compiler/latest/linux", "bin/clang"
+            os.environ.get("ONEAPI_ROOT"), "compiler/latest/linux/bin/clang"
         )
     if IS_WIN:
         compiler = os.path.join(
-            os.environ.get("ONEAPI_ROOT"), "compiler/latest/windows", "bin/clang.exe"
+            os.environ.get("ONEAPI_ROOT"), "compiler\\latest\\windows\\bin\\clang.exe"
         )
+    compiler = shlex.quote(compiler)
```

Comment:
https://docs.python.org/dev/library/shlex.html#shlex.quote The documentation states that this method will not work with Windows and opens up possiblity of the very issue this is trying to solve (command injection vulnerability).